modules/engage-paella-player/src/main/paella-opencast/plugins/es.upv.paella.opencast.downloadsButtonPlugin/localization/da-DK.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
If you introduce new localization files, you need to add them to `.crowdin.yaml`

Reply:
The crowdin configuration currently match the new plugin location